src/main/scala/akka/persistence/jdbc/query/scaladsl/JdbcReadJournal.scala

```diff
@@ -93,7 +93,10 @@ class JdbcReadJournal(config: Config)(implicit val system: ExtendedActorSystem)
 
   // Started lazily to prevent the actor for querying the db if no eventsByTag queries are used
   private[query] lazy val journalSequenceActor = system.actorOf(
-    JournalSequenceActor.props(readJournalDao, readJournalConfig.journalSequenceRetrievalConfiguration), "akka-persistence-jdbc-journal-sequence-actor")
+    JournalSequenceActor.props(readJournalDao, readJournalConfig.journalSequenceRetrievalConfiguration),
+    "akka-persistence-jdbc-journal-sequence-actor"
+  )
```

Comment:
Scalariform was changing this line and doing something weird. I added a breakline to fix it.